apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -97,47 +110,46 @@ export function InstructorAssessmentSettings({
                     id="number"
                     name="number"
                     value="${resLocals.assessment.number}"
-                    disabled
+                    ${canEdit ? '' : 'disabled'}
                   />
                   <small class="form-text text-muted">
                     The number of the assessment within the set.
                   </small>
                 </div>
                 <div class="form-group">
                   <label for="module">Module</label>
-                  <input
-                    type="text"
+                  <select
                     class="form-control"
```

Comment:
You should use the `form-select` class on `<select>` elements for correct Bootstrap styling: https://getbootstrap.com/docs/5.3/forms/select/

Reply:
Changed `form-control` class to `form-select` for both `<select>` elements in [2937e62](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/2937e62532bbb89a04b42feab9bc5e543d0b3ce0).